packages/eslint-config/index.js

```diff
@@ -20,6 +28,7 @@ module.exports = {
     },
   ],
   rules: {
+    "tailwindcss/classnames-order": "off", // Disable ordering in favor of prettier plugin
     "@typescript-eslint/semi": "error",
```

Comment:
heh we can actually delete the `"@typescript-eslint/semi": "error"` rule all together now, prettier owns that too